src/celeritas/ThrustConfig.hh

```diff
@@ -0,0 +1,31 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ThrustConfig.hh
```

Comment:
How about `celeritas/ext/ThrustNative.hh`? It's a wrapper around the external thrust library, and its main job (now) is to provide `thrust_native` wrappers?

(Actually I'm thinking `celeritas/ext/Thrust.device.hh` since we'll probably add more Celeritas↔︎Thrust utilities.)